src/kirby/components/modal/services/modal-closer-service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ModalCloserService {
```

Comment:
The naming of this file is not conformant with the default naming convention, eg.:
`modal-closer-service.ts` => `modal-closer.service.ts`